src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,69 @@
+import './NavBar.css';
+import { Link } from 'react-router-dom';
+import { useProductContext } from '../../contexts/products';
+//import { avatar } from '../images/avatar.jpg';
+
+const showSideBar = () => {
+
+    var sidebar = document.getElementById('sidebar');
+    var display = document.getElementById('sidebar').style.display;
+    sidebar.style.display = (display == 'block') ? 'none' : 'block';
+    if (sidebar.style.display == 'block') {
+        document.getElementById("menu-icon").style.display = 'none';
+        document.getElementById("logo").style.display = 'none';
+    } else {
+        document.getElementById("menu-icon").style.display = 'block';
+        document.getElementById("logo").style.display = 'block';
```

Comment:
you are manipulating the dom, you can show the sidebar without doing manipulating it